### DISCUSSION TITLE
Riso Grids: Exercise 3

### DISCUSSION THREAD

**Discussion by sharlagelfand** [posit-conf-2023/creative-coding#9] [Show and tell]
1. Open the file file `exercises/02-riso-grids/exercise-2-3.Rmd`
2. If needed, rerun the code to define, plot, and save your two layers
3. Use the given code to combine a "paper" layer with your two layers to create the riso print effect
4. Share your final image on this GitHub discussion

**Comment by Meghansaha** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/1f5d3d61-ddef-4363-ae0a-e56639295bcd)

**Comment by bshk93** [comment]
![riso_image_complete](https://github.com/posit-conf-2023/creative-coding/assets/65432465/e1f9e7d6-a2f7-4c06-a8fc-ac032fb4abbd)

**Comment by sshaughnessy** [comment]
![combined_layers](https://github.com/posit-conf-2023/creative-coding/assets/36434962/b39f5d6c-671f-4b76-990a-834e41976028)

**Comment by lwdozal** [comment]
![riso_img](https://github.com/posit-conf-2023/creative-coding/assets/19821171/f17db37e-c3da-4a61-b1c8-2c8e227bf35f)

**Comment by Kim-s-h** [comment]
![ex3](https://github.com/posit-conf-2023/creative-coding/assets/22897443/5a74674f-a898-4c59-9b85-f4e30e163937)

**Comment by datatoolbox** [comment]
![](https://b8bf4af053f14e97b22366a3e7a61f2e.app.posit.cloud/file_show?path=%2Fcloud%2Fproject%2Fexercises%2F04-riso-grids%2Friso_output.png)

**Comment by thomaspara** [comment]
![stuff3](https://github.com/posit-conf-2023/creative-coding/assets/10082327/94cc07e0-5894-439c-99b2-1e83a71cb4cc)
![stuff3 (1)](https://github.com/posit-conf-2023/creative-coding/assets/10082327/ce28ab53-a732-4142-8189-0bc57a6e6448)

**Comment by natalieoshea** [comment]
![final](https://github.com/posit-conf-2023/creative-coding/assets/10425075/8f9f73a3-9823-4688-9477-866e19b12d65)

**Comment by MHanken** [comment]
![riso_image](https://github.com/posit-conf-2023/creative-coding/assets/49319823/6e53235b-ac27-43fc-a532-92879ffb9a9a)

**Comment by shaunmwilkinson** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/145362966/c855cffe-fb7e-452e-b7e2-d462a982b05f)

**Comment by datatoolbox** [comment]
![riso-1](https://github.com/posit-conf-2023/creative-coding/assets/5366795/8dabb558-9cdb-421c-9b8e-8d44f78b124f)

**Comment by roBertus31** [comment]
![finalImg](https://github.com/posit-conf-2023/creative-coding/assets/25853405/4027885c-62af-49ed-a6dd-fb8a423c1657)

**Comment by iamciera** [comment]
![preview (8)](https://github.com/posit-conf-2023/creative-coding/assets/1732390/56e66767-b0d8-4e0a-831a-4db374664feb)
